packages/flutter_tools/templates/list_detail_app/test/implementation_test.dart.test.tmpl

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
```

Comment:
Do you still need all the `*.png.golden.tmpl` files? What are those used for now?

Reply:
Oops, nope! Thanks.